lib/ApiRequestor.php

```diff
@@ -145,16 +145,26 @@ private static function _defaultHeaders($apiKey)
 
         $langVersion = phpversion();
         $uname = php_uname();
-        $curlVersion = curl_version();
+
+        $httplib = 'curl not installed';
+        $ssllib = 'unknown';
+
+        if (function_exists('curl_version'))
+        {
```

Comment:
Nit, but I think most of the conditional curly braces (as opposed to the ones for function) are on the same line. Mind changing over to that here for consistency?